### DISCUSSION TITLE
Referencing existing functions in custom function

### DISCUSSION THREAD

**Discussion by jeffrod1** [handsontable/hyperformula#1349] [API Questions]
Hello,

I was wondering if it is, or would be, possible to reference an existing function inside a custom function? For example, I would like to call the existing MATCH function inside my custom function and use the returned index for further actions. I am essentially trying to write a LOOKUP function since there isn't one currently built-in and would like to use MATCH to start. 

Also this may be a different issue/question but I originally attempted to use the code inside of `doMatch()` inside my custom lookup function. i was able to access `this.columnSearch` but `this.rowSearch` did not give me anything. Not sure if that should be exposed or not. Thanks for your time!

**Comment by AMBudnik** [comment]
Hi @jeffrod1 

I see that the conversation/ticket is marked as solved. Do you still need any help here?

**Reply by jeffrod1** [reply]
No I figured out another way to accomplish what I was trying to do, but it would be nice to still have the ability to piggyback off an existing function if that doesn't already exist.

**Comment by sequba** [comment]
@jeffrod1 Calling a built-in function from a custom functions is technically possible but I wouldn't recommend that. Take a look at the [comment I made in another discussion](https://github.com/handsontable/hyperformula/discussions/1142#discussioncomment-4677898)
